scipy/special/meson.build

```diff
@@ -331,11 +331,7 @@ py3.extension_module('_ufuncs',
   ],
   c_args: [cython_c_args, Wno_maybe_uninitialized],
   include_directories: ['../_lib', '../_build_utils/src'],
-  dependencies: [
-    lapack,
-    npymath_lib,
```

Comment:
This must still be needed right? You were replacing it with the new subproject, but that seems to have gone missing from this PR now.